libvisual-plugins/plugins/actor/gstreamer/actor_gstreamer.c

```diff
@@ -287,7 +314,7 @@ static void act_gstreamer_render (VisPluginData *plugin, VisVideo *video, VisAud
     g_mutex_unlock (priv->mutex);
 }
 
-static void handle_sink_data (GstElement *sink, GstBuffer *buffer, GstPad *pad, GStreamerPrivate* priv)
+static void handle_sink_handhoff (GstElement *sink, GstBuffer *buffer, GstPad *pad, GStreamerPrivate* priv)
```

Comment:
Oops, "handoff" is mispelled as "handhoff".

Reply:
@kaixiong oops :smiley: Fixed! :+1: